configuration/configuration.go

```diff
@@ -501,4 +501,4 @@ var defaultKeycloakTesUser2Name = "testuser2"
 var defaultKeycloakTesUser2Secret = "testuser2"
 
 // Keycloak URL to be used in dev mode. Can be overridden by setting up keycloak.url
-var devModeKeycloakURL = "http://sso.demo.almighty.io"
+var devModeKeycloakURL = "http://sso.prod-preview.openshift.io"
```

Comment:
To be clear. This is the default URL for dev mode only. We use relative URLs by default in the "production" mode.
We could add more details to the comment if it's not clear.